packages/js/create-block-template/webpack.config.js.mustache

```diff
@@ -0,0 +1,94 @@
+const path = require( 'path' );
+const defaultConfig = require( '@wordpress/scripts/config/webpack.config' );
+const WooCommerceDependencyExtractionWebpackPlugin = require( '@wordpress/dependency-extraction-webpack-plugin' );
+const MiniCssExtractPlugin = require( 'mini-css-extract-plugin' );
+
+const wcDepMap = {
+	'@woocommerce/blocks-checkout': [ 'wc', 'blocksCheckout' ],
+	'@woocommerce/blocks-registry': [ 'wc', 'wcBlocksRegistry' ],
+	'@woocommerce/settings': [ 'wc', 'wcSettings' ],
+};
+
+const wcHandleMap = {
+	'@woocommerce/blocks-registry': 'wc-blocks-registry',
+	'@woocommerce/blocks-checkout': 'wc-blocks-checkout',
+	'@woocommerce/settings': 'wc-settings',
+};
+
+const requestToExternal = ( request ) => {
+	if ( wcDepMap[ request ] ) {
+		return wcDepMap[ request ];
+	}
+};
+
+const requestToHandle = ( request ) => {
+	if ( wcHandleMap[ request ] ) {
+		return wcHandleMap[ request ];
+	}
+};
+
+// Remove SASS rule from the default config so we can define our own.
+const defaultRules = defaultConfig.module.rules.filter( ( rule ) => {
+	return String( rule.test ) !== String( /\.(sc|sa)ss$/ );
+} );
+
+module.exports = {
+	...defaultConfig,
+	module: {
+		...defaultConfig.module,
+		rules: [
+			...defaultRules,
+			{
+				test: /\.(sc|sa)ss$/,
+				exclude: /node_modules/,
+				use: [
+					MiniCssExtractPlugin.loader,
+					{ loader: 'css-loader', options: { importLoaders: 1 } },
+					{
+						loader: 'sass-loader',
+						options: {
+							sassOptions: {
+								includePaths: [ 'src/css' ],
+							},
+							additionalData: ( content, loaderContext ) => {
+								const {
+									resourcePath,
+									rootContext,
+								} = loaderContext;
+								const relativePath = path.relative(
+									rootContext,
+									resourcePath
+								);
+
+								if (
+									relativePath.startsWith( 'src/css/' )
+								) {
+									return content;
+								}
+
+								// Add code here to prepend to all .scss/.sass files.
+								return (
+									'@import "_colors"; ' +
+									content
+								);
+							},
+						},
+					},
+				],
+			},
+		],
+	},
+	plugins: [
+		...defaultConfig.plugins.filter(
+			( plugin ) =>
+				plugin.constructor.name !== 'DependencyExtractionWebpackPlugin'
+		),
+		new WooCommerceDependencyExtractionWebpackPlugin( {
+			requestToExternal,
+			requestToHandle,
+		} ),
```

Comment:
`requestToExternal` and  `requestToHandle` can be removed so they don't overwrite the defaults.